### PR TITLE
admin user info

### PR DESCRIPTION
**Pull request by ghost**
## Description
admin user info
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: resources/views/admin/user/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <!-- Main content -->
+    <section class="content">
+      <div class="row">
+        <!-- left column -->
+        <div class="col-md-12">
+          <!-- general form elements -->
+          <div class="box box-primary">
+            <!-- /.box-header -->
+            <!-- form start -->
+            <form method="POST" action="" enctype="multipart/form-data">
```
Comment (at-tuanphan) [inline resources/views/admin/user/show.blade.php]: đâu có input file đâu mà dùng cái `enctype="multipart/form-data"` em

File: resources/views/admin/user/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                  <label for="exampleInputEmail">{{ __('user.admin.edit.email') }}</label>
+                  <input type="email" class="form-control" disabled name="email" value="{{ $user->email }}">
+                </div>
+                <div class="form-group">
+                  <label for="exampleInputRole">{{ __('user.admin.edit.address') }}</label>
+                  <input type="text" class="form-control" disabled name="address" value="{{ $user->userInfo->address }}">
+                </div>
+                <div class="form-group">
+                  <label for="exampleInputRole">{{ __('user.admin.show.address_shipping') }}</label>
+                  @if (count($user->shippings))
```
Comment (at-tuanphan) [inline resources/views/admin/user/show.blade.php]: trong foreach nó có check empty nên ko cần cái `if` đâu em

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**